src/SFML/Graphics/Shape.cpp

```diff
@@ -137,6 +136,56 @@ FloatRect Shape::getLocalBounds() const
 }
 
 
+////////////////////////////////////////////////////////////
+Vector2f Shape::getGeometricCenter() const
```

Comment:
This implementation is still not just before `getLocalBounds` implementation, contrary to the order in the class definition.

Reply:
Odd... I could have sworn I moved it... but, no. :-) Fixed now.